site/content/docs/main/restore-reference.md

```diff
@@ -169,6 +169,39 @@ data:
   # class name.
   <old-storage-class>: <new-storage-class>
 ```
+### Changing Pod/Deployment/Statefulsets/Daemonset/Replicaset/Replicationcontroller/Job/Cronjob Image Repositories  
+
+Velero can change the image repositories of pod/deployment/statefulsets/daemonset/replicaset/replicationcontroller/job/cronjob during restores. To configure a image repositoriy mapping, create a config map in the Velero namespace like the following:
```

Comment:
Everything but statefulsets is in the singular form above, so I'd change that one to "statefulset" to be consistent here.